index.php

```diff
@@ -2,7 +2,7 @@
 $filename = 'STOCK_RT.json';
 $filepath = 'bin/'.$filename;
 $theTicker = 'ANET';
-$tzOffset = '04:00'; // EDT 4, EST 5
+$tzOffset = '05:00'; // EDT 4, EST 5
 
```

Comment:
would like to see this done dynamically but given how much you hate timezones I will allow this

Reply:
I agree, this is a little disappointing, I will continue to improve this.